### ISSUE TITLE
Display connection errors on login page

### ISSUE THREAD

**Issue by ririko5834** [SamuelTS/MongoDB-PHP-GUI#21]
I think that when you type MongoDB login details it should try to connect to MongoDB. If the connection was not successful, display an error on the login page and don't let the user go to the dashboard.

**Comment by ririko5834** [comment]
Now you can fill fake not working mongodb data, and it will give you access to the dashboard.

Like this:
![image](https://user-images.githubusercontent.com/64729293/134467437-e6764ceb-cbe3-4525-a6ba-04fac95f41fd.png)

![image](https://user-images.githubusercontent.com/64729293/134467470-d69bb489-59f9-4e58-8eef-c3a1d22a6aa5.png)
When you go to visualize database, error is displayed:
![image](https://user-images.githubusercontent.com/64729293/134467530-59e56c61-5088-4102-8444-5d1574afb0a1.png)


The error should be displayed on the login page.

**Comment by SamuelTS** [comment]
I agree. Connection errors need to be handled earlier.

**Comment by ririko5834** [comment]
Yeah, then it's confusing for users why they can access the dashboard when they have provided fake connection details.
